src/proxy.rs

```diff
@@ -105,15 +114,43 @@ impl Proxy {
             outbound: self.outbound.address(),
             inbound: self.inbound.address(),
             socks5: self.socks5.address(),
+            dns_proxy: match &self.dns_proxy {
+                None => SocketAddr::new(IpAddr::V4(Ipv4Addr::UNSPECIFIED), 0),
```

Comment:
nit: should we make it an Option?

Reply:
It was an option previously, but the question came up as to why would it be an option and just not false by default.